docs/source/conf.py

```diff
@@ -291,17 +291,9 @@
 
 # -- script stuff --------------------------------------------------------
 
-import inspect
-
 
 def autodoc_skip_member(app, what, name, obj, skip, options):
```

Comment:
We can probably remove the whole "script stuff" and below now :)

Reply:
It's a part of the template so leaving it in.